docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Managing Pull Requests
+---
+
+## Pull Requests and the Community
+
+Gatsby is an open source project. We have over 2,000 contributors and so much of what makes Gatsby great is contributed by folks like you!
+
+Needless to say, we get a lot of PRs and we've been merging over a [100 contributions](https://twitter.com/kylemathews/status/1111435640581689345) every week. Yes, _every week_.
+
+Let's talk a little about how we manage pull requests in the Gatsby repo!
+
+## What is a Pull Request?
+
+In case you aren't familiar, here is how the fine folks at GitHub [define a pull request](https://help.github.com/en/articles/about-pull-requests):
+
+> Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+
+Simply put, when someone wants to contribute to Gatsby, they open a request to _pull_ their code into ours. Depending on the type of change, we like to categorise pull requests (or PRs for short) into:
+
+- Documentation
+- Code
+- Starters or Site Showcase
+- Blog posts
+
+When looking at a PR for the first time, it can help to read up on linked issues or RFCs (if there are any) to gain context on what the PR intends to add or fix.
+
+> 💡If there isn't already an issue that describes the problem a PR is trying to solve, it can be helpful to open one. This helps in sharing context and can be very valuable in the future.
+
+Each kind of PR requires a different set of checks from us before they are merged in!
+
+Let's go over them real quick.
+
+### Documentation
+
+We typically look for the following in PRs that add documentation:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
+
+### Code
+
+For PRs that add code (whether a feature or fix), we look for the following:
+
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
+- Code Quality — while it isn't good to nit pick, reasonable changes that improve readability are great to point out.
```

Comment:
Would "Code readability" fit better (instead of Code Quality)?

And reason for that is the code is being read much more often than it's being written, so making sure it's easily digestible is big priority

Reply:
I don't think code readability is what we should strive for. I think the "Code Quality" label is good, but the description should be changed. How about:

> focus on reasonable changes that will likely make code maintenance, comprehension, or correctness improved or more obvious

(tweak as needed!)